pandas/io/common.py

```diff
@@ -186,7 +190,12 @@ def get_filepath_or_buffer(filepath_or_buffer, encoding=None,
     ----------
     filepath_or_buffer : a url, filepath (str, py.path.local or pathlib.Path),
                          or buffer
+            supports 'https://username:password@fqdn.com:port/aaa.csv'
     encoding : the encoding to use to decode py3 bytes, default is 'utf-8'
+    compression:
+    auth: (str,str), default None. (username, password) for HTTP(s) basic auth
+    verify_ssl: Default True. If False, allow self signed and invalid SSL
+                 certificates for https
 
```

Comment:
1) Why is the `compression` field empty?

2) The formatting for `auth` and `verify_ssl` should be patched.  The general format is the following:

~~~python
<var_name> : <data_type>, <defaults>
     < Description>
~~~